src/Bookings.js

```diff
@@ -12,7 +12,8 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults results={FakeBookings} />
```

Comment:
This passes the `results` prop, but the `SearchResults` component doesn't actually read that prop - instead it separately loads its own copy of `FakeBookings`. Can you change `SearchResults` so that it uses this prop, and doesn't itself import `FakeBookings`?